streams/test-utils/src/main/java/org/apache/kafka/streams/TopologyTestDriver.java

```diff
@@ -478,6 +479,7 @@ private void setupTask(final StreamsConfig streamsConfig,
                     mockWallClockTime,
                     streamsConfig,
                     logContext,
+                    createAdminClient(processorTopology.storeToChangelogTopic()),
```

Comment:
This is the only thing I am wondering about. Why not just pass `null`?

Reply:
After thinking a bit more I agree with you now: I removed the mock admin client and instead just pass a mock register (since the register function may still be called)